src/cudaautotune/CUDACore/ExecutionConfiguration.h

```diff
@@ -0,0 +1,42 @@
+#ifndef HeterogeneousCore_CUDAUtilities_interface_ExecutionConfiguration_h
+#define HeterogeneousCore_CUDAUtilities_interface_ExecutionConfiguration_h
+
+#include <fstream>
+
+namespace cms {
+  namespace cuda {
+
+    class ExecutionConfiguration {
```

Comment:
Currently this class appears to be used only as a "namespace", because the objects have no state.

On the other hand, if I understood correctly, the launch parameters are read from the file for each kernel for each event. Would it be feasible to read each file only once in some way?